GitHub/1SettingUpGit.md

```diff
@@ -1,92 +1,96 @@
 
-At this point, you should have a GitHub account setup and functioning. If not, do head over [here][1] and sign yourself up! <br> <br>
+_NOTE: Make sure to use the same email address that you signed in to GitHub with!_
 
+## Github Desktop
 
-### Configuring Git:
+If you want to work with Git locally and you're worried about the command line experience, you can initially download and use the GitHub Desktop interface that provides most of the essential features of GitHub. Download it and follow the onscreen prompts.
 
-#### Configure Git CLI:
+_NOTE: Git has to be installed already for GitHub Desktop to work_
 
-To begin using GitHub from your local machine, you need to set a username and email address to identify yourself while making commits and what not. To do so, we have to run a couple of commands. <br>
+Download Github Desktop: https://desktop.github.com
+(Download the appropriate version of Github Desktop for your operating system.)
 
-*Username :* ``` git config --global user.name "FIRST_NAME LAST_NAME"``` <br> <br>
-*E-Mail ID:* ``` git config --global user.email "MY_NAME@example.com" ``` <br> <br> 
+<!-- Is the configuration step for GitHub Desktop really necessary? The whole process is quite user-friendly -->
 
-#### Configure Git for Github Desktop:
+### Configure Git for Github Desktop:
 
 Github Desktop uses the email address you set in your local Git configuration to connect the commits you make with your account on Github.
 
-#### Steps to be followed on Mac:
+### Steps to be followed on Mac:
 
 1. Sign in to Github.
 2. On the upper-right corner of the page, click on your profile photo.
 3. Select Settings from the Drop-down that appears at this point.
    <!-- photo 1. https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/configuring-git-for-github-desktop -->
-4. Under Settings, click on emails. 
+4. Under Settings, click on emails.
    <!-- photo 2 same link as in the comment above -->
 5. Copy the email address you'd like to use in your local Git configuration.
-6. Open GitHub Desktop and in the menu bar, using the drop down menu, click on Preferences. 
-    <!-- picture 3 same comment mac setup-->
+6. Open GitHub Desktop and in the menu bar, using the drop down menu, click on Preferences.
+<!-- picture 3 same comment mac setup-->
 7. In the Preferences window, click Git.
-    <!-- picture 4 same comment -->
+<!-- picture 4 same comment -->
 8. In the Name field, type the name you'd like to use for your Git configuration.
-    <!-- picture 4 same link-->
+<!-- picture 4 same link-->
 9. In the Email field, paste the email address you copied from your GitHub account settings.
-    <!-- picture 5 same link -->
+<!-- picture 5 same link -->
 10. Click Save.
 <!-- picture 6 same link -->
 
-<!--IS THIS PART NECESSARY? HOW IS IT DIFFERENT FROM macOS ? --> 
-#### Steps to be followed on Windows:
+<!--IS THIS PART NECESSARY? HOW IS IT DIFFERENT FROM macOS ? -->
```

Comment:
Not sure. @sridhar-5, justify having this step? How different is the macOS process?

Reply:
it's pretty simple in a Mac if we are installing through 'home-brew'. just one step or two if the home-brew is not setup.

Reply:
sorry for the late reply though.